### ISSUE TITLE
Orphaned preview cards

### ISSUE THREAD

**Issue by angristan** [tootsuite/mastodon#13179]
I have 141 961 files in my `preview_cards` folder.

However running

```sql
SELECT count (*) FROM "public"."preview_cards" WHERE "image_file_name" IS NOT NULL;
```

Results in `72134`.

If I'm not mistaken, that would mean half of the files are not referenced in the database, and thus orphaned?

**Comment by Gargron** [comment]
Fixed in #13316

**Comment by angristan** [comment]
Awesome @Gargron, thank you so much!
